Tests/FluentAssertions.Specs/Collections/GenericCollectionAssertionOfStringSpecs.cs

```diff
@@ -1970,5 +1970,37 @@ public void When_string_collection_does_not_satisfy_all_inspectors_it_should_thr
         }
 
         #endregion
+
+        #region AllSatisfy
```

Comment:
🔧 Instead of a region, please use a nested class. We [recently](https://github.com/fluentassertions/fluentassertions/discussions/1551#discussioncomment-2021661) started to adopt that principle.